en/deploy/README.md

```diff
@@ -1,246 +1,304 @@
+```python
+import os
+import sys
 
-## Deploy to Heroku!
+path = '/home/edith/my-first-blog'   # this should have your username in
+if path not in sys.path:
+    sys.path.append(path)
 
-That was a lot of configuration and installing, right? But you only need to do that once! Now you can deploy!
+os.environ['DJANGO_SETTINGS_MODULE'] = 'mysite.settings'
 
-When you ran `heroku create`, it automatically added the Heroku remote for our app to our repository. Now we can do a simple git push to deploy our application:
+from django.core.wsgi import get_wsgi_application
+from whitenoise.django import DjangoWhiteNoise
+application = DjangoWhiteNoise(get_wsgi_application())
+```
 
-    $ git push heroku master
+> **Note** don't forget to substitute in your own username instead of "edith"
 
-> __Note__: This will probably produce a *lot* of output the first time you run it, as Heroku compiles and installs psycopg. You'll know it's succeeded if you see something like `https://yourapplicationname.herokuapp.com/ deployed to Heroku` near the end of the output.
+This file's job is to tell PythonAnywhere where our web app lives (so make sure to substitute your own username for "edith" above) and what the Django settings file's name is.  It also sets up the "whitenoise" static files tool.
 
-## Visit your application
+Hit **Save**, and then go back to the **Web** tab.  
 
-You’ve deployed your code to Heroku, and specified the process types in a `Procfile` (we chose a `web` process type earlier).
-We can now tell Heroku to start this `web process`.
+We're all done!  Hit the big green **Reload** button and you'll be able to go view your application.  You'll find a link to it at the top of the page.
 
-To do that, run the following command:
 
-    $ heroku ps:scale web=1
+## Debugging tips
 
-This tells Heroku to run just one instance of our `web` process. Since our blog application is quite simple, we don't need too much power and so it's fine to run just one process. It's possible to ask Heroku to run more processes (by the way, Heroku calls these processes "Dynos" so don't be surprised if you see this term) but it will no longer be free.
+If you see an error when you try to visit your site, the first place to look for some debugging info is in your **error log** -- you'll find a link to this on the PythonAnywhere web tab.  See if there are any error messages in there.  The most recent ones are at the bottom.   Common problems include
 
-We can now visit the app in our browser with `heroku open`.
+- forgetting one of the steps we did in the console: creating the virtualenv, installing django into it, running collectstatic, initialising the database
+- making a mistake in the virtualenv path on the web tab -- there will usually be a little red error message on there, if there is a problem
+- making a mistake in the WSGI configuration file -- did you get the path to your my-first-blog folder right?
```

Comment:
"did you get the path to your my-first-blog folder right?" > "Did you correctly set the `path` variable that points to your my-first-blog folder?"